### PR TITLE
Modal confirm

### PR DESCRIPTION
**Pull request by abea**
I'm imagining the triggering component would kick off the modal, then execute its action on the 'confirm' event.

### CODE REVIEW

File: src/components/modal-confirm/AposModalConfirm.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  },
+  data() {
+    return {
+      modal: {
+        title: '',
+        active: false,
+        type: 'overlay',
+        showModal: false
+      },
+      lastSelected: null
```
Comment (stuartromanek) [inline src/components/modal-confirm/AposModalConfirm.vue:76]: unused

File: src/components/modal-confirm/AposModalConfirm.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+/deep/ .apos-modal__body {
+  padding: 60px;
+}
+
+.apos-confirm__icon {
+  width: 60px;
+  height: 60px;
+}
+// .apos-confirm__heading
```
Comment (stuartromanek) [inline src/components/modal-confirm/AposModalConfirm.vue:118]: cleanup

File: src/components/modal/AposModal.vue [hunk 3]
```diff
@@ -75,6 +75,10 @@ export default {
   computed: {
     id() {
       const rand = (Math.floor(Math.random() * Math.floor(10000)));
+
+      if (!this.modal.title) {
+        return 'rand';
```
Comment (stuartromanek) [inline src/components/modal/AposModal.vue:80]: ?

Comment (abea) [reply src/components/modal/AposModal.vue:80]: `rand`

**Pull request merged**